src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.html

```diff
@@ -1,200 +1,215 @@
-<div class="oeb">	
-<form-message></form-message>
-<app-badge-legend *ngIf="showLegend" (closed)="closeLegend()"></app-badge-legend>
-
-<form [formGroup]="badgeClassForm.rawControl" #formElem (ngSubmit)="onSubmit()" novalidate>
-	<!-- Badge Category Panel -->
-	<div class="section tw-text-oebblack">
-		<div class="tw-flex tw-flex-col tw-gap-2">
-			<div class="tw-flex tw-flex-col md:tw-flex-row tw-gap-4 md:tw-gap-8">
-				<h2 hlmH2 class="tw-font-bold">1. {{ 'CreateBadge.chooseBadgeCategory' | translate }}</h2>
-				<oeb-select
-					class="tw-w-[280px]"
-					ariaLabel="chooseABadgeCategory"
-					[control]="badgeClassForm.rawControlMap.badge_category"
-					[placeholder]="chooseABadgeCategory"
-					[optionMap]="categoryOptions"
-					[errorMessage]="{
-						required: chooseABadgeCategory
-					}"
-					[autofocus]="true"
-					noTopMargin="true"
-				></oeb-select>
-			</div>
-			<div class="tw-italic tw-gap-2 md:tw-pl-8">
-				<div class="tw-flex tw-py-2">
-					<p hlmP size="sm" class="tw-text-purple">{{ 'Badge.competency' | translate }}-Badges&nbsp; </p>
-					<p hlmP size="sm" class="md:tw-hidden"> {{ 'CreateBadge.competencyBadgeShortInfo' | translate }}.</p>
```

Comment:
There seems to be quite a lot of unnecessary reformatting. Maybe avoid that for the future, since it makes spotting actual changes harder and imo doesn't provide any advantage, the way it was done here.

Reply:
We'll be applying prettier to all files on the main branch on tuesday next week, so this will not be a problem from there on